mobile/src/people/avatar.tsx

```diff
@@ -12,18 +12,23 @@ const styles = StyleSheet.create({
     },
     outerFrame: {
         borderColor: '#2FAFCC',
-        borderWidth: 2
     },
     image: {
         borderColor: '#fff',
-        borderWidth: 2,
         flex: 1
+    },
+    default: {
+        height: 200,
```

Comment:
why height and width have default values? They are supposed to be taken from the actual size. BorderWidth / inner border only things that can be overriden

Reply:
we'd also discussed it previously, these defaults for cases if no size was specified.